src/color-picker/color-picker.scss

```diff
@@ -45,17 +61,29 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb
   width: $iui-l;
   cursor: pointer;
   border-radius: $iui-color-swatch-border-radius;
-  background-color: var(--iui-color-swatch-background);
   margin-bottom: $iui-xs;
   margin-right: $iui-xs;
-
+  position: relative;
+  background-position: 0 0, ($iui-m * 0.5) ($iui-m * 0.5);
+  background-size: $iui-m $iui-m;
+  @include themed {
+    background-image: $iui-transparency;
+  }
   @supports (gap: $iui-s) {
     margin-bottom: 0;
     margin-right: 0;
   }
 
-  @include themed {
-    box-shadow: $iui-inset-box-shadow;
+  &::after {
+    content: '';
+    position: absolute;
+    width: inherit;
+    height: inherit;
+    border-radius: inherit;
+    background-color: var(--iui-color-swatch-background);
```

Comment:
Could we "flip" the approach so that background is set normally in the swatch, and `::after` is used for the checkered pattern?

Currently this component supports IE by setting `background-color` as an inline style (in HTML/JSX), which would not be achievable if you move the background in a pseudo-element.

Reply:
At first I tried applying the background color on the swatch and the checkered background on the pseudo element & using `isolation: isolate` and `z-index` but it wasn't working.

So now what I did was apply the background color on the swatch, then override it with the checkered background.  The pseudo element then inherits the background color.  Does that work for IE?

Reply:
ok, seems to work
![image](https://user-images.githubusercontent.com/9084735/154516392-18b78567-17df-4be8-8ee7-bf9954c52f37.png)

the checkered pattern is absent but there's no regression

Reply:
Interesting… ok, I've removed the variable.

Reply:
You could have it in a mixin and then `@include iui-transparency`.

Also while you're at it, might as well change those hover/focus variables to use `var(--` instead of `t()`

Reply:
Created `@mixin iui-transparent-background` & replaced `t()` with `var(--` on `$iui-inset-box-shadow`, `$iui-active-box-shadow`, & `$iui-hover-box-shadow`.